client/public/index.html

```diff
@@ -2,7 +2,12 @@
 <html lang="en">
```

Comment:
I am not entirely sure why you need an index.html file here, when you are using react for building your app. the index.js file is all you need and this file is technically replacing the index.html file.

Reply:
That just belongs to create-react-app webpack. It´s nothing, I create myself but it´s necessary. It´s the output of webpack/ create-react-app